### PR TITLE
purposes in call for invibes

### PR DESCRIPTION
**Pull request by invibes**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by invibes** [comment]
@rcheptanariu please review it before anyone else!

File: modules/invibesBidAdapter.js [hunk 1]
```diff
@@ -443,7 +438,6 @@ invibes.Uid = {
   }
 };
 
-let cookieDomain;
 invibes.getCookie = function (name) {
```
Comment (robertrmartinez) [inline modules/invibesBidAdapter.js:441]: for reading cookies please use the storage manager

Comment (invibes) [reply modules/invibesBidAdapter.js:443]: change request was implemented please review for merge asap :)

**Comment by invibes** [comment]
change request was implemented please review for merge asap :)

**Comment by invibes** [comment]
@robertrmartinez please review asap

**Pull request merged**